velox/core/PlanNode.cpp

```diff
@@ -1825,6 +1825,102 @@ PlanNodePtr NestedLoopJoinNode::create(
       outputType);
 }
 
+// static
+const JoinType SpatialJoinNode::kDefaultJoinType = JoinType::kInner;
+// static
+const TypedExprPtr SpatialJoinNode::kDefaultJoinCondition = nullptr;
```

Comment:
`inline static` can be used since C++17 

Example: 
https://cppreference.com/w/cpp/language/static.html
`inline static const std::string class_name{"X"}; // Likewise`


But these can be even better with `static constexpr` and avoid runtime constants at all